ml/Host.cc

```diff
@@ -423,10 +469,12 @@ void DetectableHost::detectOnce() {
     time_t Before = now_realtime_sec();
     time_t After = Before - (WindowLength * updateEvery());
     DB.insertAnomaly("AD1", 1, getUUID(), After, Before, JsonResult.dump(4));
+
 }
 
 void DetectableHost::detect() {
     std::this_thread::sleep_for(Seconds{10});
+    AnomalyBitCounterWindow = Cfg.SaveAnomalyPercentageEvery;
```

Comment:
We should use an in-member initializer for the `AnomalyBitCounterWindow` field by changing 

`size_t AnomalyBitCounterWindow;` to `size_t AnomalyBitCounterWindow{Cfg.SaveAnomalyPercentageEvery};`

We can then remove this line.

Reply:
OK, but sometimes I avoid initializers to make sure the default constructor of the trivial types is called, as is much cheaper. but you are right in this case.